Source/PeleLMeX_Advance.cpp

```diff
@@ -120,6 +120,13 @@ PeleLM::Advance(int is_initIter)
     computeSootSource(AmrOldTime, m_dt);
   }
 #endif
+#ifdef PELELM_USE_RAD
+  if (do_rad_solve) {
+    BL_PROFILE_VAR("PeleLM::advance::rad", PLM_RAD);
+    computeRadSource(AmrOldTime, m_dt);
```

Comment:
Not necessarily for this PR: if I follow correctly, we are treating Radiation (and soot and spray) explicitly, 1st order, and decoupled from other physics. Should we recompute these source terms using the new state during each SDC iteration?

Reply:
It depends on the time scale. The current implementation should work (for the radiation) for most combustion applications. However, for more extreme applications, e.g. Sandia has a case studying the nanosecond-scale spark ignition process, you are right we need to update it within the SDC iterations. I agree with you it would be great if we have that option, but it might not be necessary for this PR.